app/scripts/clean_data.py

```diff
@@ -6,8 +6,23 @@
 import os
 import shutil
 import sys
-from ConfigParser import SafeConfigParser
 
+# Python 2/3 compatibility
+try:
+    from ConfigParser import SafeConfigParser
+except ImportError:
+    from configparser import SafeConfigParser
+
+try:
+    dict.iteritems
+except AttributeError:
+    # Python 3
+    def iteritems(d):
+        return iter(d.items())
+else:
+    # Python 2
+    def iteritems(d):
+        return d.iteritems()
```

Comment:
`six` has helpers for both of these. Guess you added six after this?

Also further down in the patch.

Reply:
This one is on purpose. 

Unlike the main script (`tmx_products.py`), this one is called directly from bash, as stand-alone, and I don't want to force having `six` as dependency.

Reply:
On second thoughts, I think it makes sense to make a bash script, assume the virtual env is available, and run this script within it.